cli/test/smokehouse/test-definitions/a11y.js

```diff
@@ -552,6 +552,10 @@ const expectations = {
           ],
         },
       },
+      'html-xml-lang-mismatch': {
+        score: null,
+        scoreDisplayMode: 'notApplicable',
+      },
```

Comment:
In order to test this, we would need to add `lang` and `xml:lang` attributes to the `<html>` tag? And that would force the `html-has-lang` audit to pass?

Reply:
Yep! I figured that this is still useful because I wasn't sure if `html-xml-lang-mismatch` also checked for the existence + validity of the `lang` attribute -- as in, whether this was `html-has-lang` + checking `xml:lang`, which the docs [seemed to imply](https://dequeuniversity.com/rules/axe/4.7/html-xml-lang-mismatch#:~:text=The%20HTML%20document%20must%20contain%20a%20valid%20lang%20attribute%20or%20must%20correspond%20to%20a%20valid%20lang%20code). But given that this is `notApplicable`, seems like that's not the case, and this solely focuses on `xml:lang`.